### ISSUE TITLE
Published single-exe doesn't want to start

### ISSUE THREAD

**Issue by Lakritzator** [dotnet/core-setup#7173]
The error can be found in the event log and is the following:

```
Description: A .NET Core application failed.
Application: Greenshot-x64.exe
Path: C:\Users\robin\Downloads\Greenshot-x64.exe
Message: The application to execute does not exist: 'C:\Users\robin\Downloads\Greenshot.dll'.
```

dotnet --info
```
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100-preview8-012985
 Commit:    2fe132f3a6

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18362
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-preview8-012985\

Host (useful for support):
  Version: 3.0.0-preview8-27910-04
  Commit:  7a43bc1550

.NET Core SDKs installed:
  2.1.700 [C:\Program Files\dotnet\sdk]
  2.1.701 [C:\Program Files\dotnet\sdk]
  3.0.100-preview8-012985 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview8.19359.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview8-27910-04 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0-preview8-27910-04 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

using dotnet publish with the following options:
-f netcoreapp3.0 -c Release /p:PublishSingleFile=true /p:RuntimeIdentifier=win-x64

Tried different repositories and bits (x86 / x64), all the same effect.

I'm assuming this is an issue with not finding the target (tmo) directory?

**Comment by jeffschwMSFT** [comment]
Fixed with https://github.com/dotnet/sdk/pull/3411

**Comment by elinor-fung** [comment]
@Lakritzator I think you will need a newer build than `3.0.100-preview8-013116`. Unfortunately, based on how the different dotnet repos come together, the fix in dotnet/sdk had to flow through a couple repos to make it to the official full build. The fix should be in `3.0.100-preview8-013133` and higher.

**Comment by Lakritzator** [comment]
@elinor-fung thanks for updating this, and now things are working again (besides some other issues).

I keep repeating myself: it's REALLY hard to be able to detect if a certain issue is fixed in a SDK version, there are not release notes or ways to see if a fix is available.
